index.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const parseArgs = (
+  argv = process.argv.slice(require.main ? 2 : 1),
```

Comment:
we might want to extend this further to handle environments like electron, I'm not sure of the best approach. Here's [what we do in yargs](https://github.com/yargs/yargs/blob/master/lib/utils/process-argv.ts).